rest_framework/fields.py

```diff
@@ -1204,6 +1204,8 @@ def to_representation(self, value):
         )
 
         if output_format.lower() == ISO_8601:
+            if (isinstance(value, str)):
```

Comment:
Sure, I will then update both `DateField` & `TimeField` to use `six.string_types`